Makefile

```diff
@@ -10,6 +10,12 @@ REGISTRY_NAMESPACE ?=
 build: $(SOURCES) ## Build binary from source
 	go build -i -ldflags="-s -w" -o _out/hyperconverged-cluster-operator ./cmd/manager
 
+install: build
```

Comment:
If you add a comment `install: build ## Help text here` makes it so `make help` will tell someone what it does.

Reply:
Done

Reply:
actually I can create an extra branch for the integration work, and once it works I can create a combined PR.